data/pg/works/expense/LabourCharges.json

```diff
@@ -0,0 +1,193 @@
+{
+    "tenantId": "pg",
+    "moduleName": "calculator",
+    "LaborCharges": [
+         {
+             "id": 1,
+             "service": "works.wages",
+             "skill": "UNSKILLED.CARPENTER",
+             "gender": "female",
+             "active": true,
+             "amount": 150,
+             "unit": "day",
+             "effectiveFrom": 24234234234234,
+             "effectiveTo": null
+         },
+         {
+            "id": 2,
+            "service": "works.wages",
+            "code": "UNSKILLED.MULIA",
+            "gender": "male",
+            "active": true,
+            "amount": 10,
+            "unit": "day",
+            "effectiveFrom": 24234234234234,
+            "effectiveTo": null
+         },
+         {
+            "id": "2",
+            "service": "works.wages",
+            "code": "UNSKILLED.MULIA",
+            "gender": "female",
+            "active": true,
+            "amount": 150,
+            "unit": "day",
+            "effectiveFrom": 24234234234234,
+            "effectiveTo": null
+         },
+         {
+             "id": "3",
+             "service": "works.wages",
+             "code": "UNSKILLED.SCAVENGER",
+             "gender": "all",
+             "active": true,
+             "amount": 150,
+             "unit": "day",
+             "effectiveFrom": 24234234234234,
+             "effectiveTo": null
+         },
+         {
+             "id": "4",
+             "service": "works.wages",
+             "code": "UNSKILLED.CHAIN_MAN",
+             "gender": "all",
+             "active": true,
+             "amount": 150,
+             "unit": "day",
+             "effectiveFrom": 24234234234234,
+             "effectiveTo": null
+         },
+         {
+             "id": "5",
+             "service": "works.wages",
+             "code": "SEMI_SKILLED",
```

Comment:
if we define this way master this will have an impact in individual service, 
currently in skills array we have type and level so if code has only semiskilled then type will be sent as null since type is mandatory.